src/components/TokenWarningCard/index.tsx

```diff
@@ -94,48 +63,82 @@ export default function TokenWarningCard({ token, ...rest }: TokenWarningCardPro
-      </Row>
+      <AutoRow gap="6px">
+        <AutoColumn gap="24px">
+          <CurrencyLogo currency={token} size={'16px'} />
+          <div> </div>
+        </AutoColumn>
+        <AutoColumn gap="10px" justify="flex-start">
+          <TYPE.main>
+            {token && token.name && token.symbol && token.name !== token.symbol
+              ? `${token.name} (${token.symbol})`
+              : token.name || token.symbol}
+          </TYPE.main>
+          <ExternalLink style={{ fontWeight: 400 }} href={getEtherscanLink(chainId, token.address, 'token')}>
+            <TYPE.blue> (View on Etherscan)</TYPE.blue>
+          </ExternalLink>
+        </AutoColumn>
+      </AutoRow>
     </Wrapper>
   )
 }
 
-const WarningContainer = styled.div`
-  max-width: 420px;
-  width: 100%;
-  padding-left: 1rem;
-  padding-right: 1rem;
-`
-
-export function TokenWarningCards({ currencies }: { currencies: { [field in Field]?: Currency } }) {
+export function TokenWarningCards({
+  currencies,
+  dismissedToken0,
+  dismissedToken1,
+  dismissToken0,
+  dismissToken1
+}: {
+  currencies: { [field in Field]?: Currency }
+  dismissedToken0: boolean
+  dismissedToken1: boolean
+  dismissToken0: (() => void) | null
+  dismissToken1: (() => void) | null
+}) {
   return (
     <WarningContainer>
-      {Object.keys(currencies).map(field =>
-        currencies[field] instanceof Token ? (
-          <TokenWarningCard style={{ marginBottom: 14 }} key={field} token={currencies[field]} />
-        ) : null
-      )}
+      <AutoColumn gap="lg">
+        <AutoRow gap="6px">
+          <AlertTriangle stroke="#F82D3A" />
+          <TYPE.main color={'#F82D3A'}>Token imported</TYPE.main>
+        </AutoRow>
+        <TYPE.body color={'#F82D3A'}>
+          Anyone can create and name any ERC20 token on Ethereum, including creating fake versions of existing tokens
+          and tokens that claim to represent projects that do not have a token.
+        </TYPE.body>
+        <TYPE.body color={'#F82D3A'}>
+          Similar to Etherscan, this site can load arbitrary tokens via token addresses. Please do your own research
+          before interacting with any ERC20 token.
+        </TYPE.body>
+        {Object.keys(currencies).map(field => {
+          return currencies[field] instanceof Token ? (
+            <TokenWarningCard
+              key={field}
+              token={currencies[field]}
+              dismissed={field === Field.INPUT ? dismissedToken0 : dismissedToken1}
+            />
+          ) : null
+        })}
+        <RowBetween>
+          <div />
+          <ButtonError
+            error={true}
+            width={'140px'}
+            padding="0.5rem 1rem"
+            style={{
+              borderRadius: '10px'
+            }}
+            onClick={() => {
+              dismissToken0 && dismissToken0()
+              dismissToken1 && dismissToken1()
+            }}
+          >
+            <TYPE.body color="white">I understand</TYPE.body>
```

Comment:
should this be like text1 or something?

Reply:
I use white because we actually want it to be white for both light and dark mode (text colors switch from black to white)